execution_workflows/APAtrap/README.md

```diff
@@ -0,0 +1,96 @@
+# APAtrap
+The APAtrap R package does the following three steps: 
+1. Refines annotated 3'UTR and identifies novel 3' UTRs and 3' UTR extensions
+2. Aims to identify all potential APA (alternative polyadenylation) sites
+3. Detects genes with differential APA site usage between conditions by leveraging 
+   the resolution of RNA-seq data
+
+The [paper](https://academic.oup.com/bioinformatics/article/34/11/1841/4816794) is titled APAtrap: identification and quantification of 
+alternative polyadenylation sites from RNA-seq data <br>
+The [application download](https://sourceforge.net/projects/apatrap/files/) is free to download
+and [user manual](https://sourceforge.net/p/apatrap/wiki/User%20Manual/) was used as a referrence
+to create the nextflow pipeline flow of this module
+
+## Running APAtrap workflow
+
+## Input & pre-processing
+Required files are to be specified in the input `samplesheet_example_files.csv`. Each row in the sample sheet has four
+columns:
+
+- condition: name of the condition (e.g Control)
+- sample: unique name of the sample for logs (e.g control_replicate1)
+- bam: BAM input file for the sample 
+- bai: BAI index file for sample's bam input
+
+It is important to name samples of the same condition with the exact condition name under the condition
+column in the sample sheet since samples are grouped per condition to be processed in the differential step. In addition,
+sample names should be unique.
+
+To run APAtrap with test data provided for APAeval, check the path to APAtrap with `pwd` and replace 
+the `path_to` in samplesheet_example_files.csv with the path from the `pwd` command. 
+
+When using your own data and input file instead of the provided test data and `samplesheet_example_files.csv`, make sure to include in the 
+input file you are using the absolute path to the file, with the four column names following the column
+names in `samplesheet_example_files.csv`.
+
+### Running with Docker or Singularity
+## Docker
+This workflow uses docker containers. To run with docker, make sure that docker is installed and running 
+(e.g. to ensure docker is running, run the command `docker --help` and a help message should be printed).
+Additionally, make sure that line 49 in Apatrap/nextflow.config file `docker.enabled=true` is uncommented while line
+51 `singularity.enabled=true` is commented out.
+
+## Singularity
+To run with singularity, comment out line 49 in Apatrap/nextflow.config file `docker.enabled=true` and make sure that line
+51 `singularity.enabled=true` is uncommented.
+
+### Parameters
+Parameters used to run APAtrap are specified in conf/modules.config file. 
+Parameters relevant to the workflow itself are:
+- `run_differentail` - set to false to obtain identification and quantification challenge outputs, or set to true
```

Comment:
```suggestion
- `run_differential` - set to false to obtain identification and quantification challenge outputs, or set to true
```
Just to fix the typo so it matches exactly with the parameter defined in `conf/modules.config`